### PR TITLE
Add AutoField primary key in test models

### PR DESCRIPTION
**Pull request by tyzhnenko**
Moved to pytest
Set default python to 3.9
Add AutoField primary key to avoid Django 3.2+ warnings

### CODE REVIEW

**Comment by tyzhnenko** [comment]
Close #58

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ django_picklefield.egg-info/*
 build/
 .coverage
 .tox
+.DS_Store
```
Comment (charettes) [inline .gitignore:8]: You should add this line to your own global `.gitignore` file as it has nothing to do with this project; it's specific to your OS.

```suggestion
```

File: tests/models.py [hunk 2]
```diff
@@ -10,21 +10,21 @@
 D2 = {1: 2, 2: 4, 3: 6, 4: 8, 5: 10}
 
 
-class TestCopyDataType(str):
+class FakeCopyDataType(str):
```
Comment (charettes) [inline tests/models.py:13]: What's the purpose of these name changes? They seem to be adding a lot of noise for no reason.

File: tox.ini [hunk 3]
```diff
@@ -5,40 +5,46 @@ envlist =
...
 basepython =
     py35: python3.5
     py36: python3.6
     py37: python3.7
     py38: python3.8
+    py39: python3.9
 usedevelop = true
 commands =
-    {envpython} -R -Wonce {envbindir}/coverage run --branch -m django test -v2 --settings=tests.settings {posargs}
+    {envpython} -R -Wonce {envbindir}/coverage run --branch -m pytest --ds tests.settings tests/tests.py {posargs}
```
Comment (charettes) [inline tox.ini:29]: What's the purpose of switching to `pytest` and requiring two now dependencies? I think the default Django test runner should be fine for this project.

**Review by charettes — COMMENTED** [review]
I think a better path forward would be to simply add `DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'` in `tests/settings.py`?

I'll try to get the project Django 3.2+ compatible this weekend and switch off TravisCI to Github actions.

**Comment by charettes** [comment]
Support was confirmed in the latest `master` and `3.1.0` was uploaded with appropriate classifiers on Pypi.

**Pull request closed without merging**